app/src/main/res/values-v29/styles.xml

```diff
@@ -12,5 +12,6 @@
     <style name="Hedvig.Theme.KeyGear">
         <item name="android:navigationBarColor">@android:color/transparent</item>
         <item name="android:statusBarColor">@android:color/transparent</item>
+        <item name="android:windowLightStatusBar">false</item>
```

Comment:
`Hedvig.Theme.KeyGear` is used by other screens, does this interfere with their looks?